### PR TITLE
pdfstream v0.4.4

### PR DESCRIPTION
**Pull request by st3107**
New release.

https://github.com/xpdAcq/PDFstream/releases/tag/0.4.4

### CODE REVIEW

**Review by mrakitin — COMMENTED** [review]
Looks good. I wonder why our bot did not detect this change. Needs investigation. It looks like it's not as good with the GitHub releases as with the PyPI ones. Should we change it to use https://pypi.org/project/pdfstream/0.4.4/#files instead?

**Comment by st3107** [comment]
Yes. Let's try using pypi as the source.

**Comment by st3107** [comment]
Do you know what is the file template of the pypi source file?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "0.4.4" %}
+{% set sha256 = "d8a934446020e064171a3142a7e724b75c61ef6044356e96371a7ae1927bddb9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (mrakitin) [inline recipe/meta.yaml:10]: Looks right, same as in https://github.com/nsls-ii-forge/bluesky-feedstock/blob/14759c5e5837ed50db9625f26b229ed3065ecbcd/recipe/meta.yaml#L10.

**Review by mrakitin — APPROVED** [review]

**Pull request merged**